kong/llm/drivers/llama2.lua

```diff
@@ -108,10 +108,10 @@ end
 local function to_raw(request_table, model)
   local messages = {}
   messages.parameters = {}
-  messages.parameters.max_new_tokens = model.options and model.options.max_tokens
-  messages.parameters.top_p = model.options and model.options.top_p or 1.0
-  messages.parameters.top_k = model.options and model.options.top_k or 40
-  messages.parameters.temperature = model.options and model.options.temperature
+  messages.parameters.max_new_tokens = request_table.max_tokens or (model.options and model.options.max_tokens)
+  messages.parameters.top_p = request_table.top_p or (model.options and model.options.top_p)
+  messages.parameters.top_k = request_table.top_k or (model.options and model.options.top_k)
+  messages.parameters.temperature = request_table.temperature or (model.options and model.options.temperature)
```

Comment:
This seems being yet another pattern of merging `request_table` and `model.options`, a unification across drivers would make sense.